cdk/domino_cdk/config/eks.py

```diff
@@ -30,12 +30,17 @@ class NodegroupBase:
         key_name: some-key-pair - Pre-existing AWS key pair to configure for instances in the nodegorup
         min_size: 1 - Minimum node count for nodegroup. Can't be 0 on managed nodegroups.
         max_size: 10 - Maximum limit for node count in node gorup
-        machine_image: AMI and/or user_data script for nodes in nodegroup. When overriding AMI, you MUST
-                       setup your custom AMI or your user_data script _MANUALLY_ to join the cluster.
-                       Additionally, options that use user_data (ie ssm_agent) are no longer valid.
-                       machine_image:
-                         ami_id: ami-123abc
-                         user_data: ...
+        ami_id: ami-123abc - AMI to use for nodegroup, empty/null will use the default EKS AMI, which
+                             is the default. When specifying an AMI, you MUST specify a custom user_data
```

Comment:
"the default EKS AMI, which is the default" -> "the default EKS AMI"

Reply:
I updated the phrasing.